tst/EnergyPlus/unit/FanCoilUnits.unit.cc

```diff
@@ -1054,44 +1054,53 @@ namespace EnergyPlus {
 
 		// expect full flow and meet capacity
 		EXPECT_NEAR( QZnReq, QUnitOut, 5.0 );
-		EXPECT_NEAR( Node( 1 ).MassFlowRate, FanCoil( 1 ).FanAirVolFlow, 0.0000000001 );
+		EXPECT_NEAR( Node( 1 ).MassFlowRate, FanCoil( 1 ).MaxAirMassFlow, 0.0000000001 );
```

Comment:
Coincidentally, FanAirVolFlow and FC MaxAirMassFlow were the same in this unit test. Corrected variable so the comparison doesn't look suspect.